src/domain/notifications/index.ts

```diff
@@ -8,3 +10,137 @@ export const NotificationType = {
   OnchainPayment: "onchain_payment",
   LnInvoicePaid: "paid-invoice",
 } as const
+
+export const NotificationChannel = {
+  Push: "push",
+} as const
+
+export const GaloyNotificationCategories = {
+  Payments: "Payments" as NotificationCategory,
+  Balance: "Balance" as NotificationCategory,
+  AdminPushNotification: "AdminPushNotification" as NotificationCategory,
+} as const
+
+export const checkedToNotificationCategory = (
+  type: string,
```

Comment:
reading this make me feel like `type` can be a keyword conflicting with js base language. maybe we could use notificationType here?

Reply:
Will change the name to `notificationCategory` as we already use `notificationType` elsewhere.